pkg/kube/pod.go

```diff
@@ -235,37 +236,68 @@ func GetPodLogs(ctx context.Context, cli kubernetes.Interface, namespace, name s
 	return string(bytes), nil
 }
 
+// getErrorWithLogs fetches logs from pod and constructs error containing last ten lines of log and specified error message
+func getErrorWithLogs(ctx context.Context, cli kubernetes.Interface, namespace, podName string, err error, errorMessage string) error {
+	r, logErr := StreamPodLogs(ctx, cli, namespace, podName)
+	if logErr != nil {
+		return errors.Wrapf(logErr, "Failed to fetch logs from the pod")
+	}
+	defer r.Close()
+
+	// Grab last log lines and put them to an error
+	lt := NewLogTail(10)
```

Comment:
There are chances that the actual error message is not included in the last 10 lines right? Or we are assuming that it will always be?

Reply:
I suppose that it should be there, but of course there is a chance that after logging an error, some other information will be logged, but frankly, I've never seen such scenario.

Reply:
Discussed offline. Keeping this solution since we don't have better options here.